dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -42,6 +42,58 @@ def test_command_template():
         assert ' --preload mymodule' in cluster._command_template
 
 
+def test_shebang_settings():
```

Comment:
You should use parametrized test as the one below for all this!

Reply:
really cool!!!!